referenceframe/frame_system.go

```diff
@@ -15,6 +18,9 @@ import (
 // World is the string "world", but made into an exported constant.
 const World = "world"
 
+// defaultPointDensity is the default value for the spatialmath.ToPoints conversion method.
```

Comment:
Comment on this should specify that this value will cause the default value defined in `spatialmath` to be used. The variable name should probably also do the same.

Reply:
ok, I updated the comment. I am unsure what to change the constant's name to. 'defaultSpatialMathPointDensity` seems a bit lengthy to me.